pkg/users/account-details.js

```diff
@@ -355,3 +347,126 @@
         </Page>
     );
 }
+
+export const AccountGroupsSelect = ({ name, groups, setError }) => {
+    const [isOpenGroup, setIsOpenGroup] = useState(false);
+    const [selected, setSelected] = useState();
+    const [primaryGroupName, setPrimaryGroupName] = useState();
+    const [history, setHistory] = useState([]);
+    const previousValue = useRef(null);
```

Comment:
These 6 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17996-20221212-195940-2c392d36-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)